README.md

```diff
@@ -29,23 +28,23 @@ Take a look at the [getting started
 guide](https://github.com/syncthing/syncthing/wiki/Getting-Started).
 
 There are a few examples for keeping syncthing running in the background
-on your system in [the etc directory](https://github.com/syncthing/syncthing/blob/master/etc).
+on your system in [the /etc directory](https://github.com/syncthing/syncthing/blob/master/etc).
```

Comment:
This is probably not right, as that seems like the global `/etc` at the root, while the intention is to point to the `etc` dir in the repo.